### PR TITLE
editing tm4 admin guide

### PR DESCRIPTION
**Pull request by xamanu**
Just a bit more editing. More to come tomorrow.

### CODE REVIEW

File: _posts/en/0500-10-24-tm-admin_en.md [hunk 1]
```diff
@@ -13,30 +13,34 @@ category: coordination
...
+    -  [Helpful hints - how you should fill in all these fields](/en/coordination/tm-admin/#helpful-hints)
+    -  [Publish - do not forget or you never get work done](/en/coordination/tm-admin/#publish)
 
-The Tasking Manager is the essential tool for all kind of groups to coordinate mapping on OpenStreetMap. It helps you to conduct a mapathon, manage a humanitarian mapping initiatives, or you want to create mapping tasks for students. The application divides an area of interest into manageable geographic chunks, that can be completed rapidly and collaboratively. This approach allows the distribution of tasks to many individual mappers, while reducing possible editing conflicts. 
+The Tasking Manager is the essential tool for all kind of groups to coordinate mapping on OpenStreetMap. It helps you to conduct a mapathon, manage a humanitarian mapping initiative, or whenever you want to create mapping tasks for students. The application divides an area of interest into manageable geographic chunks, that can be completed rapidly and collaboratively. This approach allows the distribution of tasks to many individual mappers, while reducing possible editing conflicts.
```
Comment (bopercival-hot) [inline _posts/en/0500-10-24-tm-admin_en.md:34]: <cmd-g> Suggestion: The Tasking Manager is the essential tool to coordinate mapping on OpenStreetMap. It supports humanitarian mapping initiatives, conducting mapathons or creating mapping tasks for students. The application divides an area of interest into manageable geographic chunks that can be completed rapidly and collaboratively. This approach allows the distribution of tasks to many individual mappers, while reducing possible editing conflicts.

File: _posts/en/0500-10-24-tm-admin_en.md [hunk 2]
```diff
@@ -13,30 +13,34 @@ category: coordination
...
+The Tasking Manager is the essential tool for all kind of groups to coordinate mapping on OpenStreetMap. It helps you to conduct a mapathon, manage a humanitarian mapping initiative, or whenever you want to create mapping tasks for students. The application divides an area of interest into manageable geographic chunks, that can be completed rapidly and collaboratively. This approach allows the distribution of tasks to many individual mappers, while reducing possible editing conflicts. 
 
-The Software further supports mapping accuracy and data quality by providing a consistent set of instructions for your mappers (e.g. 'map all roads and buildings'). In short, the Tasking Manager is how you set up and direct the workflow for open mapping activities. This module describes the basic administration of the  Tasking Manager for successful mapping events.
+The Software further supports mapping accuracy and good data quality by providing a consistent set of instructions for your mappers (e.g. 'map all roads and buildings'). In short, the Tasking Manager is how you set up and direct the workflow for open mapping activities.
```
Comment (bopercival-hot) [inline _posts/en/0500-10-24-tm-admin_en.md:35]: Suggestion: 
The Software encourages accurate and quality mapping data by providing a consistent set of instructions for your mappers (e.g. 'map all roads and buildings'). In short, the Tasking Manager is how you set up and direct the workflow for open mapping activities.

File: _posts/en/0500-10-24-tm-admin_en.md [hunk 3]
```diff
@@ -13,30 +13,34 @@ category: coordination
...
 
-## Access the Tasking Manager
+## Login and access
 
-Access is given to the Tasking Manager by users authorizing themselves through the [OpenStreetMap website](https://www.openstreetmap.org). In order to login to the Tasking Manager an OpenStreetMap (OSM) account is needed. You can visit the [HOT Tasking Manager](http://tasks.hotosm.org), or any other community instance of your preferance. There click `Sign up` and it redirects you to OpenStreetMap.org to create your account. Or click `Log in` if you have already one, and in the pop-up you can give the Tasking Manager limited access to your OSM account. 
+Access is given by users authorizing themselves through the [OpenStreetMap website](https://www.openstreetmap.org). In order to login to the Tasking Manager an OpenStreetMap (OSM) account is needed. You can visit the [HOT Tasking Manager](http://tasks.hotosm.org), or any other community instance of your preferance. There click `Sign up` and it redirects you to OpenStreetMap.org to create your account. Or click `Log in` if you have already one, and in the pop-up you can give the Tasking Manager limited access to your OSM account.
```
Comment (bopercival-hot) [inline _posts/en/0500-10-24-tm-admin_en.md:42]: Suggestion: 
In order to login to the Tasking Manager an OpenStreetMap (OSM) account is needed. You can visit the [HOT Tasking Manager](http://tasks.hotosm.org), or any other community instance of your preferance. There click `Sign up` and it redirects you to OpenStreetMap.org to create your account. Or click `Log in` if you have already one, and in the pop-up you can give the Tasking Manager limited access to your OSM account.

File: _posts/en/0500-10-24-tm-admin_en.md [hunk 4]
```diff
@@ -13,30 +13,34 @@ category: coordination
...
+Access is given by users authorizing themselves through the [OpenStreetMap website](https://www.openstreetmap.org). In order to login to the Tasking Manager an OpenStreetMap (OSM) account is needed. You can visit the [HOT Tasking Manager](http://tasks.hotosm.org), or any other community instance of your preferance. There click `Sign up` and it redirects you to OpenStreetMap.org to create your account. Or click `Log in` if you have already one, and in the pop-up you can give the Tasking Manager limited access to your OSM account. 
 
-### Handling permissions
+## Permissions
 
 The most basic user level is the one of a **mapper**. Mappers are able to login to the Tasking Manager, and they can find and select a mapping project to work on. All of their functionality is described in the [LearnOSM Tasking Manager Mapper Guide](/en/coordination/tm-user/). Note that some projects, such as those in draft state (i.e. unpublished), or restricted to certain teams, are not visible to regular mappers.
```
Comment (bopercival-hot) [inline _posts/en/0500-10-24-tm-admin_en.md:45]: The most basic user level is **mapper**. Mappers are able to login to the Tasking Manager to find and select a mapping project to work on. All **mapper** functionality is described in the [LearnOSM Tasking Manager Mapper Guide](/en/coordination/tm-user/). Note that some projects, such as those in draft state (i.e. unpublished), or restricted to certain teams, are not visible to regular mappers

File: _posts/en/0500-10-24-tm-admin_en.md [hunk 5]
```diff
@@ -123,9 +126,9 @@ Learn more about how to add descriptions and instructions to the project and get
 
 You can be much more precise to draw the area of interest with a geospatial desktop application like JOSM or QGIS, than with the slippy web map inside the Tasking Manager. And you can also determine special task shapes and sizes beforehand.
```
Comment (bopercival-hot) [inline _posts/en/0500-10-24-tm-admin_en.md:127]: Suggestion: 
Geospatial desktop applications like JOSM or QGIS allow mappers to more precisely draw the areas of interest, compared to the slippy web map inside the Tasking Manager. You can also determine special task shapes and sizes beforehand.

File: _posts/en/0500-10-24-tm-admin_en.md [hunk 6]
```diff
@@ -137,11 +140,11 @@ After uploading a file to the Tasking Manager you can select if you want to defi
 1. `Square Grid`, with the functionality described above to define tasks.
 2. `Arbitrary Geometries`, coming from the file you just uploaded.
 
-#### Define arbitrary geometries
+**Define arbitrary geometries**
 
-If you uploaded a file to define your project area, that file can also contain the shapes for the individual tasks. E.g. if the area of interest consists of one polygon, the project will have just one task. This is generally not needed and should only be needed for specific needs. An import of roads project might need custom task shapes or the output from an mapping area reduction process for example.
+If you uploaded a file to define your project area, that file may also contain the shapes for the individual tasks. E.g. if the area of interest consists of one polygon, the project will have just one task. This is generally not needed and should only be needed for specific needs. An import of roads project might need custom task shapes or the output from an mapping area reduction process for example.
```
Comment (bopercival-hot) [inline _posts/en/0500-10-24-tm-admin_en.md:145]: Suggestion: 
If you uploaded a file to define your project area, that file may also contain the shapes for the individual tasks. E.g. if the area of interest consists of one polygon, the project will have just one task. This should only be used in special cases and for specific reasons. For example, an import of roads project might need custom task shapes or output from a mapping area reduction process

File: _posts/en/0500-10-24-tm-admin_en.md [hunk 7]
```diff
@@ -137,11 +140,11 @@ After uploading a file to the Tasking Manager you can select if you want to defi
...
 
-Please note, arbitrary geometries can not be split.
+Please note, arbitrary geometries can not be split, and they should only be used in special cases, as they have implications on the performance of the tool.
```
Comment (bopercival-hot) [inline _posts/en/0500-10-24-tm-admin_en.md:147]: Please note, arbitrary geometries can not be split. They should only be used in special cases, as they have implications on the performance of the tool.

File: _posts/en/0500-10-24-tm-admin_en.md [hunk 8]
```diff
@@ -157,7 +160,6 @@ After a project has been initially set up and created, you have to edit its info
 - Actions - Send messages to contributors, validate and invalidate the entire project with one click.
 
 > Filling in this information carefully is an essential part of making a successfull mapping project. It is very important that the objectives of the project are clear and that all resources which the mappers should take into account are laid out here. Keep in mind, that a fair share of mappers doesn't have previous experience with OpenStreetMap, and are not well familiar with tagging guidelines.
```
Comment (bopercival-hot) [inline _posts/en/0500-10-24-tm-admin_en.md:162]: Suggestion:
> Filling in this information carefully is an essential part of a successful mapping project. It is very important project objectives and resources are clearly communicated so mappers are aware of specific and important information. Keep in mind, mappers may not have previous experience with OpenStreetMap, and will not be familiar with tagging guidelines.

File: _posts/en/0500-10-24-tm-admin_en.md [hunk 9]
```diff
@@ -157,7 +160,6 @@ After a project has been initially set up and created, you have to edit its info
...
-> 
 >  It is often advisable to confine one project to one class of objects to be mapped. If you need a basemap of an area better split it in several projects, one for the roads, one for the buildings etc. Now beginners can focus on a small class of objects while learning how to map them correctly.
```
Comment (bopercival-hot) [inline _posts/en/0500-10-24-tm-admin_en.md:163]: >  It is advisable to confine one project to one class of objects to be mapped. If you need a basemap of an area better split it in several projects, one for the roads, one for the buildings etc. Now beginners can focus on a small class of objects while learning how to map them correctly.

**Comment by michael63-osm** [comment]
@bopercival-hot   I merged your suggestions in a separate commit. Thank you for your comments and for the time spent on reading this guide!

**Pull request merged**